adapters/http/car.go

```diff
@@ -0,0 +1,126 @@
+package http
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo"
+
+	"github.com/ucladevx/BPool/interfaces"
+	"github.com/ucladevx/BPool/models"
+	"github.com/ucladevx/BPool/services"
+	"github.com/ucladevx/BPool/utils/auth"
+)
+
+type (
+	// CarService is used to handle all car CRUD operations
+	CarService interface {
+		GetAllCars(lastID string, limit, authLevel int) ([]*models.Car, error)
```

Comment:
Not saying this is bad naming, but is this not redundant to say CarService.GetAllCars? when you see that it returns a slice of cars?